### ISSUE TITLE
minZoom doesn't work

### ISSUE THREAD

**Issue by guntherkoo** [scottcheng/cropit#70]
Smaller images are imported and they are not fit to width or height like v2.0. Tried minZoom but has no effect. Here is what I have for the settings.. please help! 

```javascript
$imageCropper.cropit({
			imageBackground: true,
			exportZoom: 2,
			imageBackgroundBorderWidth: [0, 25, 0, 25],
			rejectSmallImage : false,
			minZoom: 'fill',
			onZoomDisabled: function(){
				$imageZoomBar.addClass('inactive');
			},
			onZoomEnabled: function(){
				$imageZoomBar.removeClass('inactive');
			},
			onImageError : function(err){
				console.log('cropit.onImageError ' + err.message);
			},
		});
```

**Comment by Cain-Ish** [comment]
minZoom: 'fit'

**Comment by scottcheng** [comment]
@guntherkoo Thanks for posting the issue. This is a scenario that I did not consider. I'll make some change to support it now.

**Comment by scottcheng** [comment]
Fixed in 9bdefa8bb51d6ccd4486f965dbf447f009193fb3.

**Comment by scottcheng** [comment]
@guntherkoo if you upgrade to cropit@0.4.0 and use `{ smallImage: 'stretch' }` it'll fill/fit the container, depending on `minZoom` option.
